### DISCUSSION TITLE
Windows Subsystem for Android (WSA) Version 2311.40000.1.0 has been released

### DISCUSSION THREAD

**Discussion by sn-o-w** [LSPosed/MagiskOnWSALocal#749] [General]
Windows Subsystem for Android (WSA) Version 2311.40000.1.0 has been released.

Instructions:
1. On the Windows side, download it from https://github.com/bubbles-wow/WSA-Archive/releases/download/2311.40000.1.0/MicrosoftCorporationII.WindowsSubsystemForAndroid_2311.40000.1.0_neutral_._8wekyb3d8bbwe.Msixbundle with a big thanks to @bubbles-wow for providing the file archive
2. Rename the downloaded file to **wsa-WIF.zip**
3. Delete everything inside the download folder for MagiskOnWSALocal
4. Place the **wsa-WIF.zip** file in the MagiskOnWSALocal download folder
   *** Note *** Build is broken per [[Bug] Build Error - Create system images #717](https://github.com/LSPosed/MagiskOnWSALocal/issues/717) but can be built by manually replacing contents of scripts/build.sh as seen here:
   [[Bug] Build Error - Create system images #717 (comment)](https://github.com/LSPosed/MagiskOnWSALocal/issues/717#issuecomment-1779576833)
5. In Linux, build MagiskOnWSALocal as usual and remember to choose "Insider Fast" for the WSA release type.

**Comment by hugo-lyppens** [comment]
Does anyone know whether this version 2311 restores the LAN discovery ability lost since 2306 (2305 last known working WSA version)
